pkg/certifier/scorecard/scorecardRunner_test.go

```diff
@@ -24,6 +24,9 @@ import (
 )
 
 func Test_scorecardRunner_GetScore(t *testing.T) {
+	if os.Getenv("GITHUB_EVENT_NAME") != "check_suite" || os.Getenv("GITHUB_EVENT_ACTION") != "completed" {
+		t.Skip("Skipping test: Not running on post-merge")
+	}
```

Comment:
Thanks for this. I'd rather not have GitHub Action specific env vars in the code here. It couples our tests and code to a specific environment, and breaks the abstraction.

Changing the `//go:build integration` line above to a different tag, then running this test in a separate workflow makes sense to me.

We should probably have an "expensive" or "resource constrained" set of tests that only run post-merge, as PRs get run over and over as they are updated.